packages/imperative/src/rest/src/session/doc/IPropsToPromptFor.ts

```diff
@@ -0,0 +1,8 @@
+import { ISession } from "../../..";
```

Comment:
two quick things on this file...

1. We are missing the license header
2. I think it might be good to import from the specific file (or at least some more specific location) to avoid possible circular references in the future 😋

Reply:
resolved